Cargo.toml

```diff
@@ -85,6 +85,10 @@ rev = "140b6f3"
 [patch.crates-io.usb-device]
 git = "https://github.com/rust-embedded-community/usb-device"
 
+[patch.crates-io.encdec-base]
+git = "https://github.com/quartiq/rust-encdec"
+branch = "feature/float-support"
```

Comment:
Hmm... It seems like cargo isn't picking up the right encdec-base package from this patch. It works locally, so perhaps this is a cargo version issue?